examples/gnrc_border_router/README.md

```diff
@@ -10,11 +10,11 @@ run on an Atmel SAM R21 Xplained Pro evaluation board using an external UART
 adapter for the second serial interface. However, it is feasible to run the
 example on any RIOT supported platform that offers either more than one UART or
 be equipped with an IPv6 capable network device. In this case only the Makefile
-of this application has to be slightly modified (e.g. by replacing the line
+of this application has to be slightly modified, e.g. by replacing the line
 ```
 USEMODULE += gnrc_slip
 ```
-by something like
+with something like
```

Comment:
dict.leo.org says "replace by so/sth" -> ablösen, "replace with so/sth" -> austauschen, auswechseln. So in this case "with" is probably correct.